README.md

```diff
@@ -4,4 +4,4 @@ A collection of tutorials on how to get started with Clojure using different edi
 
 * [Getting Started using Nightcode](nightcode.md)
 
-######Note that the getting started guide being used by ClojureBridge workshops is [here](https://github.com/ClojureBridge/curriculum/blob/master/outline/setup.md).
+######Note that the getting started guide being used by ClojureBridge workshops is [here](https://github.com/ClojureBridge/curriculum/blob/gh-pages/outline/setup.md).
```

Comment:
Is the URL correct? Shouldn't this be the denver-clojurebridge org's fork of curriculum?